ansible/roles_ocp_workloads/ocp4_workload_rhel_worker/readme.adoc

```diff
@@ -0,0 +1,58 @@
+= ocp4-workload-rhel-worker - Set up AWS RHEL8 worker via tasks on Bastion
+
+== Role overview
+
+* This role installs playbooks and shell scripts to deploy a RHEL8 Worker for AWS based deployments.
+
+Once this role has been executed there is a directory `/opt/rhel_worker` with the necessary files to deploy a rhel_worker.
+If `ocp4_workloads_rhel_worker_deploy_worker` is set, this workload will deploy one worker.
+You can deploy subsequent workers with the playbook.
+SSH into the bastion, change to the `/opt/rhel_worker` directory and run `./setup_rhel_worker.sh`.
+This script will execute the playbook to deploy a rhel_worker into the region specified.
+
+Currently only `eu-central-1`, `us-east-1` and `us-east-2` are supported. This is because only for these three regions the AMI for the RHEL 8 image is specified.
+
+. The role consists of the following tasks files:
+** Tasks: link:./tasks/pre_workload.yml[pre_workload.yml] - Sets up an environment for the workload deployment
+*** Debug task will print out: `pre_workload Tasks completed successfully.`
+
+** Tasks: link:./tasks/workload.yml[workload.yml] - Install the RHEL 8 Worker Playbooks onto the Bastion
+*** Debug task will print out: `workload Tasks completed successfully.`
+
+** Tasks: link:./tasks/post_workload.yml[post_workload.yml] - Used to configure the workload after deployment
+*** This role doesn't do anything here
+*** Debug task will print out: `post_workload Tasks completed successfully.`
+
+** Tasks: link:./tasks/remove_workload.yml[remove_workload.yml] - Used to delete the workload
+*** This role doesn't do anything here
+*** Debug task will print out: `remove_workload Tasks completed successfully.`
+
+== Review the defaults variable file
+
+* This file link:./defaults/main.yml[./defaults/main.yml] contains all the variables you need to define to control the deployment of your workload.
+* The variable *ocp_username* is mandatory to assign the workload to the correct OpenShift user.
```

Comment:
i don't see `ocp_username` as a mandatory variable. the variable is not used except in a `debug` task?

Reply:
readme.adoc updated
Cluster admin is assumed